### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by dubstard**
"5myetherwallet.com",
"binancecoinairdrop.com",
"cryptoairdrop.online",
"docs.pancakeswap.services",
"hiccup.pancakeswap.services",
"import.metamask.com.ecdprojects.com.au",
"metamask.best",
"metamasks.blog-update.com",
"metamaskservicesweb.com",
"metamaskwebs.com",
"pancakeswap-fin.nefft-hr.com",
"pancakeswap.finance.agusbalitour.com",
"pancakeswap.finance.airlanggarentcar.com",
"pancakeswap.finance.akarbaru.com",
"pancakeswap.finance.akhmadkholid.com",
"pancakeswap.finance.alifdashproject.com",
"pancakeswap.finance.bitminer-cloud.com",
"pancakeswap.finance.mybulusofficial.com",
"pancakeswap.finance.nusacoco.com",
"pancakeswap.finance.thevanadi.com",
"pancakeswap.finance.tiketperjalanandinas.com",
"pancakeswap.services",
"pancakeswap.themunnarholidays.com",
"pancakeswap.webcodetree.com",
"pancakeswapbot.org",
"pancakeswaps.life",
"restoremetamask.dev",
"restoremetamask.net",
"trustwallet-tokengiveaway.store",
"trustwalletcommunity--giveaway.com",
"uniswap-finance.org",
"uniswap-v2-airdrop.net",
"uniswap.tw",
"uniswapbot.net",
"uniswapshop.online",
"uniswapx3.xyz",
"v5myetherwallet.com",
"walletconnect-bridge.buymeasmoothie.xyz",

### CODE REVIEW

**Comment by dubstard** [comment]
hm, did i miss a comma again?
```
all.json:314:5 - error TS1005: ',' expected.

314     "pancakeswap.finance.alifdashproject.com",
```

File: all.json [hunk 1]
```diff
@@ -297,10 +307,25 @@
     "pancakeswajp.org",
     "pancakeswap-finance-swap-liquidity-v2.site",
     "pancakeswap-finance.live",
+    "pancakeswap.finance.agusbalitour.com",
+    "pancakeswap.finance.airlanggarentcar.com",
+    "pancakeswap.finance.akarbaru.com",
+    "pancakeswap.finance.akhmadkholid.com"
```
Comment (jacogr) [inline all.json:313]:
```suggestion
    "pancakeswap.finance.akhmadkholid.com",
```

missing `,`

**Review by jacogr — APPROVED** [review]
Thank you.

**Comment by dubstard** [comment]
Apologies for the nuisance and for overlooking. I used a rex exp to add `"` at `^` and `",` at `$`, guess i trusted it too much.

**Pull request merged**